packages/utilities/README.md

```diff
@@ -21,16 +23,98 @@ This wrapper also exposes an optional property for a custom react error boundary
 Any extra props will be passed directly to the imported module.
 
 Usage looks something like this:
+
 ```js
-import { FederationBoundary } from "@module-federation/utilities/src/react";
+import { FederationBoundary } from '@module-federation/utilities/src/react';
 
 const MyPage = () => {
   return (
     <FederationBoundary
-      dynamicImport={() => import("some_remote_host_name").then((m) => m.Component)}
-      fallback={() => import("@npm/backup").then((m) => m.Component)}
+      dynamicImport={() =>
+        import('some_remote_host_name').then((m) => m.Component)
+      }
+      fallback={() => import('@npm/backup').then((m) => m.Component)}
       customBoundary={CustomErrorBoundary}
     />
-  )
-}
+  );
+};
+```
+
+---
+
+`ImportRemote`
+
+A function which will enable dynamic imports of remotely exposed modules using the Module Federation plugin. It uses the method described in the official Webpack configuration under <a href="https://webpack.js.org/concepts/module-federation/#dynamic-remote-containers" target="_blank">Dynamic Remote Containers</a>.
+
+This function will allow you to provide a static url or an async method to retrieve a url from a configuration service.
+
+Usage looks something like this:
+
+```js
+import { importRemote } from 'module-federation-import-remote';
```

Comment:
please correct the import statement in all the related sections.

`module-federation-import-remote` doesn't exist. Since this is a README, they might think that the path really exists.

Reply:
Thanks! Fixed.